indico/modules/events/registration/controllers/api/checkin_test.py

```diff
@@ -0,0 +1,127 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2023 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+import re
+from datetime import datetime
+from pathlib import Path
+
+import pytest
+import yaml
+
+from indico.modules.events.features.util import set_feature_enabled
+from indico.util.date_time import as_utc
+
+
+pytest_plugins = ('indico.modules.events.registration.testing.fixtures', 'indico.core.oauth.testing.fixtures')
+
+
+@pytest.fixture(autouse=True)
+def configure_dummy_event(dummy_event):
+    set_feature_enabled(dummy_event, 'registration', True)
+    dummy_event.start_dt = as_utc(datetime(2023, 11, 10, 0, 0))
+    dummy_event.end_dt = as_utc(datetime(2023, 11, 12, 0, 0))
```

Comment:
I don't think this is needed anymore given that the regex replaces that (though I'd prefer to just add two more fixtures for regforms and registrations so that we don't need the scary-looking regex but up to you ;))

Reply:
The datetimes would have been somewhat easy to deal with but we still have UUIDs and IDs (in particular for regform fields) and making those consistent would be more of a hassle.

I wouldn't mind moving this into a util at some point though - at least once we want to use it in other tests.

Reply:
Ah yeah, regform fields are a pain to mock :sweat_smile: